credentials/credentials.go

```diff
@@ -38,7 +43,16 @@ func getLicenseKeySecretId(conf *config.Configuration) string {
 		return conf.LicenseKeySecretId
 	}
 
-	return defaultSecretId
+	return ""
```

Comment:
Are we sure that we want to return an empty string instead of the default secret ID in this case?